docker/docker-compose.2204.main.yaml

```diff
@@ -6,7 +6,7 @@ services:
     image: webauthn-swift:22.04-main
     build:
       args:
-        base_image: "swiftlang/swift:nightly-main-focal"
```

Comment:
@0xTim While debugging, I noticed this was still set to focal, which I _think_ is older? Should we update it here? It Doesn't seem to impact anything yet.

Also, should we add a 5.10 docker compose file? Probably in separate PR?

Reply:
> Also, should we add a 5.10 docker compose file?

When ready just @ me in the PR to add 5.10 CI.

Reply:
@dimitribouniol yeah lets do 5.10 in a new PR

Re jammy vs focal - there's not a huge amount in it but we probably should migrate if you want to do that in the 5.10 PR

Reply:
Cool, I’ll tackle this today as well.

Reply:
5.10 compose file added here: https://github.com/swift-server/webauthn-swift/pull/55

Seems like focal was just used in the `main` variant — jammy was used everywhere else including the base.